README.md

```diff
@@ -73,7 +73,7 @@ Here a configuration example:
 
 ```yaml
 m6_web_amqp:
-    sandbox: false
+    sandbox: false #optional - default false
```

Comment:
I think what @b-viguier was asking you for is to modify your configuration to be able to simulate a Amqp queue and build our own queued messages content, such as:

```yaml
sandbox:
    enabled: false # optional - default false
    messages:
        array: # a list of PHP array which represent a single Amqp message content
            - foo: 'the value of the key "foo" in my first message content'
              bar: 'another value in my first message'
            - game: 'the value of the key "games" in my second message content'
              thrones: 'another value in my second message'
        json: # a list of full path to a JSON file which represent a single Amqp message content
            - '%kernel.root_dir%/src/my_first_message.json'
            - '%kernel.root_dir%/src/AppBundle/my_second_message.json'
```

then, if you set `enabled: true`, you won't need a RabbitMQ instance running to be able to test how your consumers gather and process produced messages

Notice that, `json` key, is the same as the `array` key, except that the produced message will be an `array_decode($json, true)` of each `.json` file. 

We could also imagine, a `raw` key, which point to files left untouched (the produced message will be the exact file content).